### PR TITLE
Bugfix/characteristic column may contain space in sdrf file

### PR DESCRIPTION
**Pull request by lingyun1010**
We have un-normalised sdrf column titles for factor and characteristics

### CODE REVIEW

File: bin/load_exp_design.sh [hunk 1]
```diff
@@ -16,16 +16,21 @@ sdrf_file=${SDRF_FILE:-$3}
 
 # Reason for creating this array is to search factor value column
 # In some sdrf files this column is mentioned as "Factor Value" and in some as "FactorValue"
-FactorArray=( FactorValue "Factor Value" )
-
+FactorArray=( FactorValue "Factor Value" "Factor Value " "FactorValue ")
+CharacteristicsArray=( Characteristics "Characteristics " )
```
Comment (alfonsomunozpomer) [inline bin/load_exp_design.sh]: I’m not the designated reviewer, but I think [pattern matching](https://www.linuxjournal.com/content/pattern-matching-bash) is a more robust solution.

**Comment by alfonsomunozpomer** [comment]
Sorry I neglected this one for a while. Could you please add tests with their respective SDRF files in the `tests` directory?

**Review by alfonsomunozpomer — CHANGES_REQUESTED** [review]
All good, just missing tests to ensure SDRF files will be properly processed.

**Review by ke4 — APPROVED** [review]
LGTA 👍

**Pull request merged**